blocks/library/search/index.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Server-side rendering of the `core/search` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Renders the `core/search` block on server.
+ *
+ * Copied from get_search_form() because the core
+ * search form function doesn't allow to change markup (yet).
+
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the search form markup.
+ */
+function gutenberg_render_block_core_search( $attributes ) {
+
+	$search_form_template = locate_template( 'searchform.php' );
+	if ( '' != $search_form_template ) {
+		ob_start();
+		require( $search_form_template );
+		$form = ob_get_clean();
+	} else {
+
+		$defaults = array(
+			'label'       => _x( 'Search for:', 'label', 'gutenberg' ),
+			'placeholder' => esc_attr_x( 'Search &hellip;', 'placeholder', 'gutenberg' ),
```

Comment:
Translations shouldn't use the domain param as this file is going to be copied to core.